dlp/snippets/resources/accounts.txt

```diff
@@ -0,0 +1 @@
+My credit card number is 1234 5678 9012 3456, and my CVV is 789.
```

Comment:
To fix licence linter please add:
```
dlp/snippets/resources/accounts.txt
dlp/snippets/resources/harmless.txt
dlp/snippets/resources/test.txt
```

To `ignoreFiles` in [header-checker-lint.yml](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/7c183c00ad57d29f8b703a7cdbfd4878ea9451c5/.github/header-checker-lint.yml#L10)

Reply:
Thanks.